test/SlowTests/Cluster/ClusterTransactionTests.cs

```diff
@@ -1122,8 +1124,16 @@ public async Task ClusterTransactionRequestWithRevisions(Options options)
                     }
                 }.Initialize())
                 {
-                    // let the document with the revision to replicate
-                    Assert.True(WaitForDocument(revivedStore, "foo/bar"));
+                    try
+                    {
+                        // let the document with the revision to replicate
+                        Assert.True(WaitForDocument(revivedStore, "foo/bar"));
+                    }
+                    catch
+                    {
+                        Console.WriteLine(await CollectDebugInfo(leader, revived, leaderStore, revivedStore, options));
```

Comment:
Not sure if this will show up when running from `dotnet test`. Switch to `Output.WriteLine` instead.

Reply:
Changed
Are you sure it'll print to the console with that `Output.WriteLine`?